src/translations/en.json

```diff
@@ -4900,7 +4900,19 @@
             "set_param_queued": "The parameter change has been queued, and will be updated when the device wakes up.",
             "set_param_error": "An error occurred.",
             "parameter": "Parameter",
-            "bitmask": "Bitmask"
+            "bitmask": "Bitmask",
+            "reset_to_default": {
+              "button_label": "Reset to default configuration",
+              "dialog": {
+                "title": "Reset to default",
+                "text": "Are you sure you want to reset all configuration parameters of the device {deviceName} to their default values?",
```

Comment:
```suggestion
                "text": "All device parameters will be reset to their default values",
```

Reply:
According to the other confirmation dialogs, the text should be a description of the action, not a question.

Reply:
thanks, I updated it like this.